Ontology/Agent/Agent.json

```diff
@@ -23,6 +23,25 @@
       },
       "name": "owns"
     },
+    {
+      "@type": "Property",
+      "displayName": {
+        "en": "Custom Tags"
+      },
+      "name": "customTags",
+      "dtmi:dtdl:property:schema;2": {
+        "@type": "Map",
+        "dtmi:dtdl:property:mapKey;2": {
+          "name": "tagName",
+          "schema": "string"
+        },
+        "dtmi:dtdl:property:mapValue;2": {
+          "name": "tagValue",
+          "schema": "string"
```

Comment:
I think that the mapValue for the tags should be "boolean" such that are just marker tags instead of key-value pairs.

Reply:
@rszcodronski , don't you think it's better to make it string to make it more generic.  I am not convinced that we need a second set customProperties, but I am open as well

Reply:
I'm fine with removing customTags and customProperties if we want implementers to extend the ontology and add them in themselves. This is what Willow has already done.

What's an example of how you would use customTags where the tagValue is a string?